src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs

```diff
@@ -136,7 +136,6 @@ public async Task<IActionResult> Authorize()
                 return View(new AuthorizeViewModel
                 {
                     ApplicationName = await _applicationManager.GetLocalizedDisplayNameAsync(application),
-                    RequestId = request.RequestId,
```

Comment:
Note: `OpenIddictRequest.RequestId` is obsolete in OpenIddict 6.1.0+ (the request caching feature now uses `request_uri`) but that property wasn't used anywhere so I decided to remove it.